### PR TITLE
refactor: page header component

### PR DESCRIPTION
**Pull request by IlyesBenAmara**
This PR is a refactor of the page header component used in multiple screens

### CODE REVIEW

File: src/components/shell/PageHeader.tsx [hunk 1]
```diff
@@ -1,62 +1,84 @@
...
+  Grid,
+  IconButton,
+  Link,
+  Stack,
+  Typography
+} from '@mui/material'
 
 interface PageHeaderProps {
   breadcrumbs: Array<{ icon?: JSX.Element; title?: string; link?: string }>
+  buttonComponents?: JSX.Element[]
```
Comment (marrouchi) [inline src/components/shell/PageHeader.tsx:13]:
```suggestion
  buttons?: JSX.Element[]
```

File: src/components/shell/PageHeader.tsx [hunk 2]
```diff
@@ -1,62 +1,84 @@
...
+              >
+                {description}
+              </Typography>
+            ) : (
+              description
+            )}
+          </Grid>
+        </Grid>
+        <Grid item container spacing={2} xs="auto">
+          {buttonComponents?.map(elem => (
```
Comment (marrouchi) [inline src/components/shell/PageHeader.tsx:77]:
```suggestion
          {buttonComponents?.map(button => (
```

**Review by marrouchi — CHANGES_REQUESTED** [review]
Looks good ! Just a couple of minor changes.

**Review by marrouchi — APPROVED** [review]

**Review by Lumiere-Lodi — COMMENTED** [review]
LGTM

**Pull request merged**